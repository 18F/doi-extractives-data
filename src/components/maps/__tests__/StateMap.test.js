
import React from 'react'
import renderer from 'react-test-renderer'
import StateMap from '../StateMap'

describe('StateMap', () => {
    it('renders correctly', () => {
	const map = renderer
	      .create(<StateMap
		      ownership={true}
                      no_outline={true}
                      // offshore_regions={}
                      // states={}
		      />)
	      .toJSON()
	expect(map).toMatchSnapshot()
    })
})
