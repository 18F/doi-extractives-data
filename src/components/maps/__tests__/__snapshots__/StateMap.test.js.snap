// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateMap renders correctly 1`] = `
<div
  class="state svg-container map-container wide"
  is="eiti-tooltip-wrapper"
  style={
    Object {
      "paddingBottom": "60.15169270833333%",
    }
  }
>
  <svg
    className="states map  ownership  no-outlines "
    viewBox="22 60 936 525"
  />
</div>
`;
